docs/docs/reference/release-notes/v3.0/index.md

```diff
@@ -0,0 +1,271 @@
+
+<video controls autoplay loop>
+  <source
+    type="video/mp4"
+    src="/docs/reference/built-in-components/layouts.mp4"
+  />
+  <p>Your browser does not support the video element.</p>
+</video>
+
+There is a codemod available to help migrate from `gatsby-image`. Support is included in all sources that use `childImageSharp`, as well as out of the box support in several CMS plugins, with more coming soon.
+
+For more information, see:
+
+- [Migrating from gatsby-image to gatsby-plugin-image](/docs/reference/release-notes/image-migration-guide/)
+- [The step-by-step guide to using gatsby-plugin-image](/docs/how-to/images-and-media/using-gatsby-plugin-image/)
+- [The reference guide](/docs/reference/built-in-components/gatsby-plugin-image/)
+- [The plugin README](/plugins/gatsby-plugin-image/)
+
+## gatsby-source-wordpress@5.0.0
+
+Recently we’ve announced the brand new WordPress integration. Refer to [this blog post](/blog/wordpress-integration/)
+for full details.
+
+The originally published version of renewed `gatsby-source-wordpress` is `4.0.0`. It is
+fully compatible with Gatsby v2.
+
+For Gatsby v3 bump we’ve also bumped `gatsby-source-wordpress` to `5.0.0`. It should
+be a straight-forward update from `gatsby-source-wordpress@^4.0.0`, no additional changes
+from you are required.
+
+- Fixed a bug where adding a post type named "Filter" would break options queries [PR #29718](https://github.com/gatsbyjs/gatsby/pull/29718)
+- Fixed an html field gatsby-image regression that would prevent image transforms in certain situations [PR #29778](https://github.com/gatsbyjs/gatsby/pull/29778) and [PR #29813](https://github.com/gatsbyjs/gatsby/pull/29813)
+
+## gatsby-source-contentful@5.0.0
+
+- Migrated to the latest Contentful SDK, via [PR #29520](https://github.com/gatsbyjs/gatsby/pull/29520)
+- Compatibility with `gatsby-plugin-image`
+- Retries when downloading assets
+- Retries on network errors
+
+## gatsby-plugin-gatsby-cloud@1.0.0
+
+Gatsby Cloud now includes Hosting for all your Gatsby projects. `gatsby-plugin-gatsby-cloud` is a new plugin that you need to install when using Gatsby Cloud Hosting. It generates redirects and rewrites for your client side routes automatically and even allows you to set custom Headers on a per-file basis.
+
+For Gatsby v3 we’ve also bumped `gatsby-plugin-gatsby-cloud` to `2.0.0`. It should
+be a straight-forward update from `gatsby-plugin-gatsby-cloud@^1.0.0`, no additional changes
+from you are required.
+
+Read more about `gatsby-plugin-gatsby-cloud` at the [README](/packages/gatsby-plugin-gatsby-cloud/README.md).
+
+## Miscellaneous changes in plugins
+
+### `babel-preset-gatsby`
+
+`babel-preset-gatsby` now accepts `reactImportSource` which is passed to the underlying `@babel/preset-react` importSource field. Note that this field is only supported when `reactRuntime` is `automatic`, it is `classic` by default.
```

Comment:
Unclear: Is the "it" in the last sentence ("it is classic by default") referring to `reactRuntime` or `reactImportSource`?

```suggestion
`babel-preset-gatsby` now accepts `reactImportSource` which is passed to the underlying `@babel/preset-react` importSource field. Note that this field is only supported when `reactRuntime` is `automatic`. It is `classic` by default.
```